doc/alertmanager-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,310 @@
+          span=4,
+          format='s',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.99,
+              rate(alertmanager_silences_query_duration_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='99th Percentile'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.50,
+              rate(alertmanager_silences_query_duration_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='Median'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            rate(alertmanager_silences_query_duration_seconds_sum{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            /
+            rate(alertmanager_silences_query_duration_seconds_count{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+          ||| % $._config, legendFormat='Average'
+        ));
+
+      local silencesGossips =
+        graphPanel.new(
+          'Gossips',
+          datasource='$datasource',
+          span=4,
+          format='reqps',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('rate(alertmanager_silences_gossip_messages_propagated_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])' % $._config, legendFormat='Messages propagated'));
+
+      local silenceGcDuration =
+        graphPanel.new(
+          'Garbage Collection Duration',
+          datasource='$datasource',
+          span=4,
+          format='s',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target(
+          |||
+            rate(alertmanager_silences_gc_duration_seconds_sum{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            /
+            rate(alertmanager_silences_gc_duration_seconds_count{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+          ||| % $._config, legendFormat='Duration'
+        ));
+
+      local silenceSnaphotDuration =
+        graphPanel.new(
+          'Snapshot Duration',
+          datasource='$datasource',
+          span=4,
+          format='s',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target(
+          |||
+            rate(alertmanager_silences_snapshot_duration_seconds_sum{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            /
+            rate(alertmanager_silences_snapshot_duration_seconds_count{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+          ||| % $._config, legendFormat='Duration'
+        ));
+
+
+      local HTTPRequestRate =
+        graphPanel.new(
+          'Request Rate',
```

Comment:
The requests here are mostly UI driven, and then there is the constant traffic for scraping /metrics. Not sure if this is of enough interest for the overview dashboard.